class-tgm-plugin-activation.php

```diff
@@ -2459,7 +2466,7 @@ public function column_version( $item ) {
 		 * @since 2.2.0
 		 */
 		public function no_items() {
-			printf( wp_kses_post( __( 'No plugins to install, update or activate. <a href="%1$s">Return to the Dashboard</a>', 'tgmpa' ) ), esc_url( self_admin_url() ) );
+			echo wp_kses_post( esc_html__( 'No plugins to install, update or activate.', 'tgmpa' ) . ' <a href="' . esc_url( self_admin_url() ) . '"> ' . esc_html__( 'Return to the Dashboard', 'tgmpa' ) . '</a>' );
```

Comment:
Do it on the command line then: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html